tsconfig.build.json

```diff
@@ -1,7 +1,7 @@
 {
     "extends": "./tsconfig.base.json",
     "compilerOptions": {
-        "outDir": "out",
+        "outDir": "out/src",
```

Comment:
This file can actually be removed. The build one is never used. Compile does the same thing and is used by the packager. I use watch for development.

Reply:
If it isn't removed the fix above needs to be made or it won't work right if it is ever used.
I'm good either way though I would point out that removing it means removing a few other bits that reference it.
Well, maybe just the one task but I have to look.
Let me know if you want me to do one or the other.